modules/dmrpp_module/build_dmrpp_h4/build_dmrpp_h4.cc

```diff
@@ -51,18 +51,22 @@ using namespace build_dmrpp_util_h4;
 
 void usage() {
     const char *help = R"(
-    build_dmrpp -h: Show this help
+    build_dmrpp_h4 -h: Show this help
 
-    build_dmrpp -V: Show build versions for components that make up the program
+    build_dmrpp_h4 -V: Show build versions for components that make up the program
 
-    build_dmrpp -f <data file> -r <dmr file> [-u <href url>]: As above, but uses the DMR
-       read from the given file.
+    build_dmrpp_h4 -f <data file> -r <dmr file> [-u <href url>] [-M] [-D] [-v] [-V] [-d] 
```

Comment:
I think there should be a vertical bar between "-f <data file> -r <dmr file>" like "-f <data file> | -r <dmr file>" although I'm not 100% certain.

Reply:
No, there should be no vertical bar. Just -f -r.